website/sidebars.js

```diff
@@ -21,20 +21,20 @@ const sidebars = {
     {
       type: "doc",
       id: "intro/intro",
-      label: "A Gentle Introduction"
+      label: "A gentle introduction: Arbitrum"
```

Comment:
Eventual goal is to normalize the usage of Gentle Intros as the "informational onboarding" content type for each header section, while Quickstarts are offer "procedural onboarding".

- Details & next steps [here](https://www.notion.so/arbitrum/Dev-Docs-Site-V2-Information-architecture-onboarding-and-next-steps-9ecd42550da2401ca71ee8dbb376aeb6).
- Content types and conventions [here](https://www.notion.so/Style-and-convention-guide-Docs-dafaf8d1ce34433d88c1ef3b4b99c19c)